Kickstarter-iOS/Info.plist

```diff
@@ -85,7 +85,7 @@
 				<key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
 				<false/>
 			</dict>
-			<key>ksr.dev</key>
+			<key>ksr.test</key>
```

Comment:
Is this the local URL folks are using these days? I haven't ran the environment locally in a while 😬

Reply:
Yes, it changed a while ago. 😄